keyboards/handwired/prkl30/keymaps/default/keymap.c

```diff
@@ -0,0 +1,106 @@
+ */
+[_LOWER] = LAYOUT(
+    KC_TILD, KC_1, KC_2,   KC_3, KC_4,  KC_5, KC_6, KC_7, KC_8, KC_9,
+    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+    _______, _______,  _______, _______, _______, _______, _______, _______, _______, _______,
+    KC_0,    NO_PLUS, KC_DEL,  _______, _______, _______, NO_MINS, _______
+),
+
+/* Raise - AltGred Characters and more symbols
+ * ,------------------------------------------------------------------------------------------.
+ * |   ~  |   >  |   @  |   £  |   $  |   %  |   &  |   |  |   [  |   ]  |   ≈  | ?    | {    |
+ * |------+------+------+------+------+------+------+------+------+------+------+------|------|
+ * |      |      |      |  UP  |      |      |      |      |      |      |      |      | }    |
+ * |------+------+------+------+------+------+------+------+------+------+------+------|------'
+ * |      |   <  | LEFT | DOWN | RIGHT|             |      |      |      |      |      |
+ * `------+------+------+------+------+------+------+------+------+------+------+------'
+ */
+[_RAISE] = LAYOUT(
+    NO_TILD, NO_SECT, NO_AT, NO_PND, NO_DLR,  KC_PERC, NO_AMPR, NO_LCBR, NO_LBRC, NO_RBRC,
+    _______, _______, _______, KC_UP, _______, _______, _______, _______, _______, _______,
+    _______, NO_HALF, KC_LEFT, KC_DOWN, KC_RIGHT, _______, _______, _______, _______, _______,
+    NO_RCBR, NO_QUES, LSFT(NO_LBRC), _______, _______, LSFT(NO_RBRC), _______, _______
+),
+
+ /* FN
+ * ,------------------------------------------------------------------------------------------.
+ * |RESET |   F1 |   F2 |   F3 |   F4 |   F5 |   F6 |   F7 |   F8 |   F9 |  F10 |  F11 | F12  |
+ * |------+------+------+------+------+------+------+------+------+------+------+------|------|
+ * |      |RGB_P |RGB_HD|RGB_HI| VOL- | PREV | NEXT | VOL+ |      |      |      | PRKL |      |
+ * |------+------+------+------+------+------+------+------+------+------+------+------|------'
+ * | RGB_M|RGB_VD|RGB_VI|RGB_TG|      | PLAY/PAUSE  |      |      |      |LCA(DEL)|    |
+ * `------+------+------+------+------+------+------+------+------+------+------+------'
+ */
+[_FN] = LAYOUT(
+    RESET, KC_F1, KC_F2, KC_F3, KC_F4,  KC_F5, KC_F6, KC_F7, KC_F8, KC_F9,
+    _______, RGB_M_P, RGB_HUD, RGB_HUI, KC_VOLD, KC_MPRV, KC_MNXT, KC_VOLU, _______, _______,
+    RGB_MOD, RGB_VAD, RGB_VAI, RGB_TOG, _______, KC_MPLY, _______, _______, _______, _______,
+    KC_F10, KC_F11, KC_F12, _______, PRKL, _______, LCA(KC_DEL), _______
+),
+
+
+};
+
+void encoder_update_user(uint8_t index, bool clockwise) {
+    if (clockwise) {
+      tap_code(KC_TAB);
+    } else {
+      tap_code(KC_PGUP);
+    }
+}
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+  switch (keycode) {
+    case PRKL:
+      if (record->event.pressed) {
+        //When PRKL is pressed, print out the holy power word of our people
+        SEND_STRING("PERKELE");
```

Comment:
:rofl:

Reply:
Yea this was board was designed for the Finnish keyboard community, so we gotta have a proper macro don't we ? :D